Couldn't update ember-a11y-refocus to v4 due to preinstall script

Hello. I tried to update `ember-a11y-refocus` to `4.1.0` in a monorepo that uses yarn@v1 and CircleCI.

- Locally, I was able to run `yarn install`.
- In CI, `yarn install` failed with the error,

    ```sh
    error /home/circleci/code/node_modules/ember-a11y-refocus: Command failed.
    Exit code: 254
    Command: npx only-allow pnpm
    Arguments: 
    Directory: /home/circleci/code/node_modules/ember-a11y-refocus
    Output:
    npm ERR! code ENOENT
    npm ERR! syscall lstat
    npm ERR! path /home/circleci/.local/lib
    npm ERR! errno -2
    npm ERR! enoent ENOENT: no such file or directory, lstat '/home/circleci/.local/lib'
    npm ERR! enoent This is related to npm not being able to find a file.
    npm ERR! enoent
    ```

I'm not sure yet how CI was able to run the `preinstall` script from an external library. I'm also unsure if the issue is limited to my project, or if maybe others with `npm` or `yarn` might have encountered it.

At any rate, I was wondering if we could remove [the `preinstall` script](https://github.com/ember-a11y/ember-a11y-refocus/blob/v4.1.0/package.json#L26), since it won't be possible for contributors to switch `pnpm` to `npm` or `yarn` in the `main` branch, without a PR review and approval.

What are your thoughts?

--- a/package.json
+++ b/package.json
@@ -23,7 +23,6 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "preinstall": "npx only-allow pnpm",
     "start": "ember serve -p 0",
     "test": "npm-run-all lint test:*",
     "test:ember": "ember test",

